### ISSUE TITLE
Support Arabic Language 

### ISSUE THREAD

**Issue by abdelkrimkr** [Huanshere/VideoLingo#264]

**Comment by assassinliujie** [comment]
maybe you should ask whisperX https://github.com/m-bain/whisperX
![image](https://github.com/user-attachments/assets/7aa06f70-77b6-4033-ae32-4739f0ff3bdb)

**Comment by Huanshere** [comment]
Yes, I'm sorry. Currently, only WhisperX can resolve the hallucination problem with Whisper. For other languages, I might consider using Whisper to create a simpler version (after all, without word-level subtitles, it's difficult to segment and line-break).

**Comment by Huanshere** [comment]
https://github.com/m-bain/whisperX/blob/main/whisperx/alignment.py#L18

Seems like whisperX have supported Arabic language already, I'll try adding support later!
